### DISCUSSION TITLE
No data available custom layout

### DISCUSSION THREAD

**Discussion by N7ghtm4r3** [Ahmad-Hamwi/lazy-pagination-compose#8] [Ideas]
Hi, can be possible implement the possibility to display a custom layout such as `firstPageProgressIndicator` when there is no data available from the `PaginationState`?

**Comment by Ahmad-Hamwi** [comment]
Hey! Yes it is possible, I'll be implementing this in the near future, thank you for the reminder!

**Reply by N7ghtm4r3** [reply]
Thank you for the integration!

**Comment by Ahmad-Hamwi** [comment]
Hey I've implemented this in version 1.4.0, kindly let me know if it works out well for you. Cheers!

**Comment by N7ghtm4r3** [comment]
Yes sure! Thank you for the implementation

**Comment by N7ghtm4r3** [comment]
I tried  it and really great job! It works perfectly, thank you again!

**Reply by Ahmad-Hamwi** [reply]
Any time! Glad it worked out well for you 🎉
